mkosi/distributions/gentoo.py

```diff
@@ -211,43 +112,58 @@ def fetch_fix_stage3(self) -> None:
             else:
                 die("profile names changed upstream?")
 
+        stage3_tar_path = state.cache_dir / stage3_tar
+        cls.stage3_cache = stage3_tar_path.with_name(stage3_tar.name).with_suffix(".tmp")
+
+        user_config_path = cls.stage3_cache / USER_CONFIG_PATH
+
+        emerge_vars = {
+            "FEATURES": " ".join([
+                    # -user* are required for access to USER_CONFIG_PATH
+                    "-userfetch",
+                    "-userpriv",
+                    "-usersync",
+                    "-usersandbox",
+                    "-sandbox",
+                    "-pid-sandbox",  # for cross-compile scenarios
+                    "-network-sandbox",
+                    "parallel-install",
+                    "getbinpkg",
+                    "-candy",
+                    "noman",
+                    "nodoc",
+                    "noinfo",
```

Comment:
We shouldn't set these by default right? Only if `config.with_docs` is `True`?

Reply:
we wanted smaller image no? :)

Reply:
they're conditional now.